templates/publisher/listing.html

```diff
@@ -217,11 +217,9 @@
           </div>
 
           <div class="row">
-            <div class="col-2">
-              <label class="p-label--grid-baseline">Developer website: </label>
-            </div>
             <div class="col-8">
-              <div class="p-form-validation u-no-margin {% if field_errors and field_errors['website'] %}is-error{% endif %}">
+              <div class="p-form-validation {% if field_errors and field_errors['website'] %}is-error{% endif %}">
+                <label class="p-label--grid-baseline">Developer website: </label>
```

Comment:
`p-label--grid-baseline` as added to properly align baseline of label and input field when they were in different columns.

Reply:
done, thanks